tests/php/modules/shortcodes/test-class.instagram.php

```diff
@@ -372,4 +390,125 @@ public function test_shortcodes_instagram_non_amp( $shortcode_content ) {
 		add_filter( 'jetpack_is_amp_request', '__return_false' );
 		$this->assertNotContains( 'amp-instagram', do_shortcode( $shortcode_content ) );
 	}
+
+	/**
+	 * Test the build of a set of whitelisted parameters from a variety of inputs.
```

Comment:
```suggestion
	 * Test the build of a set of allowed parameters from a variety of inputs.
```

Reply:
Missed when I refactored the function earlier. 97d7fc97d